platform-sdk/platform-apps/tests/ConsistencyTestingTool/src/main/java/com/swirlds/demo/consistency/ConsistencyTestingToolMain.java

```diff
@@ -97,21 +84,26 @@ public void run() {
     @Override
     @NonNull
     public ConsistencyTestingToolState newStateRoot() {
-        final ConsistencyTestingToolState state = new ConsistencyTestingToolState();
-        TestingAppStateInitializer.DEFAULT.initConsensusModuleStates(state);
+        final Configuration configuration = ConfigurationBuilder.create()
```

Comment:
I wonder if there is some way to set a global config per app, so a custom `Configuration` object is not created here..